crates/libs/windows/src/core/mod.rs

```diff
@@ -124,3 +124,33 @@ macro_rules! interface_hierarchy {
 
 #[doc(hidden)]
 pub use interface_hierarchy;
+
+#[doc(hidden)]
+#[macro_export]
+macro_rules! windows_link {
+    ($library:literal, $abi:literal fn $name:ident($($arg:ident: $argty:ty),*)->$ret:ty) => (
```

Comment:
The comma between the library and abi seems a little awkward...

Reply:
I can remove it - wasn't sure if that was better/worse esthetically.